CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+
+- [#75](https://github.com/FormidableLabs/react-fast-compare/pull/75). Drop support for Node 8.
```

Comment:
I'd leave this as "drop **test** support for Node 8". As our library works untranspiled in ie11, it definitely works in lower nodes 😉